modules/microshift-manifests-override-paths.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-applications.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-manifests-override-paths_{context}"]
+= Override the list of manifest paths
+You can override the list of default paths with the following procedure.
```

Comment:
```suggestion
You can override the list of default manifest paths by using a new single path, or by using a new glob pattern for multiple files. Use the following procedure to customize your manifest paths.
```
Something like that?

Reply:
I like it. I had that as a placeholder, but I like yours better than what I had going on in my head.